client/analytics/settings/index.scss

```
@@ -11,6 +11,10 @@
 		margin-left: 15%;
 	}
 
+	@include breakpoint( '<782px' ) {
+		margin-bottom: 40px;
```

We have a SCSS variable that matches `40px` named [`$gap-largest`](https://github.com/woocommerce/woocommerce-admin/blob/master/client/stylesheets/abstracts/_variables.scss#L8), and I think it would be great to use it here. So what do you think about changing this line like this:
```suggestion
		margin-bottom: $gap-largest;
```

---

Hi Ajay, That is great. We can use that variable. I have applied the chnages and commited it.